sky/provision/azure/instance.py

```diff
@@ -101,8 +101,8 @@ def cluster_status_map(
     ) -> Dict['AzureInstanceStatus', Optional[status_lib.ClusterStatus]]:
         return {
             cls.PENDING: status_lib.ClusterStatus.INIT,
-            cls.STOPPING: status_lib.ClusterStatus.INIT,
             cls.RUNNING: status_lib.ClusterStatus.UP,
+            cls.STOPPING: status_lib.ClusterStatus.STOPPED,
```

Comment:
@yika-luo I updated this to be consistent with other clouds like AWS:
https://github.com/skypilot-org/skypilot/blob/6e5083293f0d9a9d069d51274c57f0e59e47e5ce/sky/provision/aws/instance.py#L595
Seem ok, or am I overlooking something?

Reply:
We can only stop a UP cluster then if we are in process of stopping shouldn't the status still be UP? I guess same Q applies to all other clouds @Michaelvll

Reply:
I think we want to distinguish between STOPPING and STOPPED, we also need to be able to distinguish between STOPPING and INIT or UP. For cluster termination, we don't want to have to wait for the cluster to fully terminate, but we want it to be STOPPING at least.

Reply:
Apparently Lambda and Paperspace have the same problem:
https://github.com/skypilot-org/skypilot/blob/6e5083293f0d9a9d069d51274c57f0e59e47e5ce/sky/provision/lambda_cloud/instance.py#L236
https://github.com/skypilot-org/skypilot/blob/6e5083293f0d9a9d069d51274c57f0e59e47e5ce/sky/provision/paperspace/instance.py#L294

With this PR, it can cause sky down to "fail" for these clouds since we time out waiting for the state transition.

Reply:
Having STOPPING mapped to STOPPED should be good.

Reply:
I will update lambda and paperspace as well then